machines/Makefile

```diff
@@ -286,16 +286,13 @@ ifeq ($(MODEL),driver)
 else
   ifeq ($(strip $(COMPILER)),nag)
     ifeq ($(DEBUG), TRUE)
+      ifeq ($(strip $(MACH)),hobart)
        # GCC needs to be able to link to
        # nagfor runtime to get autoconf
        # tests to work.
-       # No longer hard-coded to $(NAG), as we now have
-       # modules on goldbach. The modules on goldbach 
-       # now have $COMPILER_PATH defined, so we can 
-       # get the base path to the compiler without any 
-       # acrobatics.
-       CFLAGS += -Wl,--as-needed,--allow-shlib-undefined
-       SLIBS += -L$(COMPILER_PATH)/lib/NAG_Fortran -lf60rts
+         CFLAGS += -Wl,--as-needed,--allow-shlib-undefined
+         SLIBS += -L$(COMPILER_PATH)/lib/NAG_Fortran -lf60rts
+       endif
```

Comment:
I don't think it's a NAG 5/6 difference, because this dates back to the original NAG port. In that case it was `-lf53` instead of `-l60rts`, but I found that the new version of the flags was necessary when I initially did the NAG 6 port, which was around a year ago I think.

However, this was only needed for the MCT/mpi-serial (and maybe PIO) autoconf tests to run. With a newer MCT, and with PIO having switched to CMake, maybe that's what makes these lines unnecessary?